appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -110,7 +111,7 @@ public class Appcues internal constructor(internal val scope: AppcuesScope) {
     private val activityScreenTracking by scope.inject<ActivityScreenTracking>()
     private val deepLinkHandler by scope.inject<DeepLinkHandler>()
     private val debuggerManager by scope.inject<AppcuesDebuggerManager>()
-    private val appcuesCoroutineScope by scope.inject<AppcuesCoroutineScope>()
+    private val appcuesCoroutineScope by scope.inject<CoroutineScope>()
```

Comment:
lots of changes will be like this - where instead of resolving `AppcuesCoroutineScope` from the DI container, it will be the higher level `CoroutineScope` interface.